lib/brains/tests/test_subscription.py

```diff
@@ -56,7 +56,7 @@ def test_ok(self):
             'payment_method_token': mock.ANY,
             'plan_id': 'brick',
             'descriptor': {
-                'name': 'Mozilla*product',
+                'name': 'Mozilla',
```

Comment:
did you try a subscription in the sandbox with this? I know it sounds weird but I swear I got an error when it didn't have a ``*`` symbol.